hw/ip/spi_host/doc/checklist.md

```diff
@@ -125,17 +125,17 @@ Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Not Started |
 Testbench     | [TB_GEN_AUTOMATED][]                  | Not Started |
 Tests         | [SIM_SMOKE_TEST_PASSING][]            | Not Started |
 Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Not Started |
-Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | DONE        |
 Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Not Started |
 Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | Not Started |
 Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Not Started |
-Regression    | [FPV_REGRESSION_SETUP][]              | Not Started |
+Regression    | [FPV_REGRESSION_SETUP][]              | N/A         |
```

Comment:
we should add it to FPV regression to check the assertion in RTL. https://reports.opentitan.org/hw/top_earlgrey/formal/summary.html
Perhaps @cindychip can help on this

Reply:
ok I will change it back to not started.
@cindychip can you help me get this in?